src/DynamoCore/Graph/Workspaces/PackageDependencyInfo.cs

```diff
@@ -101,6 +101,131 @@ public override string ToString()
         }
     }
 
+    /// <summary>
+    /// Class containing info about a LocalDefinition
+    /// </summary>
+    internal class LocalDefinitionInfo: INodeLibraryDependencyInfo
+    {
+        /// <summary>
+        /// Name of the LocalDefinition
+        /// </summary>
+        public string Name { get; private set; }
+
+        /// <summary>
+        /// Path of the LocalDefinition
+        /// </summary>
+        public string Path { get; internal set; }
+
+        /// <summary>
+        /// Size of the LocalDefinition
+        /// </summary>
+        public string Size { get; internal set; }
+
+        /// <summary>
+        /// ReferenceType of the LocalDefinition
+        /// </summary>
+        public ReferenceType ReferenceType { get; internal set; }
+
+        private HashSet<Guid> nodes;
+
+        public HashSet<Guid> Nodes
```

Comment:
I dont see this field got leveraged? Did I miss anything?

Reply:
It gets called in the AddDependent method where the node is added to the list.